src/borg/testsuite/archiver.py

```diff
@@ -3186,6 +3186,11 @@ def test_with_lock(self):
         cmd = 'python3', '-c', 'import os, sys; sys.exit(42 if os.path.exists("%s") else 23)' % lock_path
         self.cmd('with-lock', self.repository_location, *cmd, fork=True, exit_code=42)
 
+    def test_with_lock_non_existent_command(self):
+        self.cmd('init', '--encryption=repokey', self.repository_location)
+        cmd = 'non_existant_command'
```

Comment:
typo: existent

Reply:
I don't see it. Looks [good](https://www.linguee.com/english-german/search?source=auto&query=existent) to me?

Reply:
3191 has it with an "a".

Reply:
Silly me...